alerts.tf

```diff
@@ -23,65 +23,48 @@ resource "newrelic_synthetics_monitor" "health_check" {
 
 # urgent conditions
 
-resource "newrelic_synthetics_alert_condition" "health_check" {
+resource "newrelic_nrql_alert_condition" "health_check" {
   count = var.service_healthcheck_url != null ? 1 : 0
 
-  policy_id = newrelic_alert_policy.urgent.id
-
-  name        = "${var.newrelic_app_name}: health check"
-  monitor_id  = newrelic_synthetics_monitor.health_check[0].id
-  runbook_url = var.runbook_url
-}
-
-resource "newrelic_nrql_alert_condition" "error_rate" {
-  count = var.alert_error_rate_enable ? 1 : 0
-
   account_id = var.newrelic_account_id
   policy_id  = newrelic_alert_policy.urgent.id
 
-  name        = "${var.newrelic_app_name}: too many sustained errors"
+  name        = "${var.newrelic_app_name}: health check"
   runbook_url = var.runbook_url
   enabled     = true
 
   aggregation_method = "event_flow"
+  aggregation_window = 60
```

Comment:
Why is only aggregation window specified for this condition?

Reply:
I'll remove it.